test/README.md

```diff
@@ -85,3 +85,18 @@ You must specify a remote host for XDebug to connect to via the `XDEBUG_REMOTE_H
 - By default npm will output some irrelevant info when a script returns a non-zero error code, as described [here](https://github.com/npm/npm/issues/8821). To prevent this pass `-s` or `--silent` to `npm run`, e.g. `npm run -s tests:unit`.
 
 - Running the entire integration test suite with XDebug enabled sometimes results in a segmentation fault. This appears to be an issue with XDebug itself.
+
+## Quick install for Developer in Ubuntu
+```
+ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
```

Comment:
These step are not the same as in the `requirement` section at the top of this file.
Should have the same instruction at both place.

in the requirement section only install docker-compose while here, install docker-ce, docker-engine and docker-compose

Reply:
The requirement section should IMHO reflect the same requirement as what is installed in the quick install section. 
And both need to be up to date with the latest docker version.